package.json

```diff
@@ -4,32 +4,32 @@
   "description": "Home to Syracuse's diverse and growing developer scene.",
   "version": "0.1.0",
   "dependencies": {
-    "@mdx-js/mdx": "^1.0.19",
-    "@mdx-js/react": "^1.0.16",
-    "babel-plugin-styled-components": "^1.10.0",
-    "gatsby": "^2.5.4",
-    "gatsby-cli": "^2.5.14",
-    "gatsby-image": "^2.1.0",
-    "gatsby-mdx": "^0.6.3",
-    "gatsby-plugin-catch-links": "^2.0.14",
-    "gatsby-plugin-google-fonts": "^1.0.0",
-    "gatsby-plugin-manifest": "^2.1.1",
-    "gatsby-plugin-react-helmet": "^3.0.12",
-    "gatsby-plugin-robots-txt": "^1.4.0",
-    "gatsby-plugin-sharp": "^2.0.37",
-    "gatsby-plugin-sitemap": "^2.1.0",
-    "gatsby-plugin-styled-components": "^3.0.7",
-    "gatsby-source-filesystem": "^2.0.36",
-    "gatsby-transformer-json": "^2.1.11",
-    "gatsby-transformer-sharp": "^2.1.19",
-    "moment": "^2.24.0",
-    "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-feather": "^1.1.6",
-    "react-helmet": "^5.2.1",
-    "styled-components": "^4.2.0",
-    "styled-reset": "^2.0.11"
+    "@mdx-js/mdx": "^1.6.21",
+    "@mdx-js/react": "^1.6.21",
+    "babel-plugin-styled-components": "> 1.10.0",
+    "gatsby": "> 2.5.4",
```

Comment:
@lannonbr is there a yarn command or something that will repin these deps for me? I just changed them so they'd get latest, but haven't figure out the best practices here... HALP!